apps/hub-app/src/components/DaoCards.tsx

```diff
@@ -11,7 +12,8 @@ const CardListBox = styled.div`
   flex-wrap: wrap;
   column-gap: 6rem;
   row-gap: 3rem;
-  .dao-card {
+  @media ${widthQuery.sm} {
```

Comment:
Oh this breaks away from the other media queries with min-width. The idea being that media queries encapsulate styles for views larger than mobile

Reply:
Switched queries here and in DAOCards to mobile first.